components/automate-chef-io/content/docs/iam-v2-api-reference.md

```diff
@@ -10,60 +10,145 @@ toc = true
+  - [Creating a Project](#creating-a-project)
+  - [Updating a Project](#updating-a-project)
+  - [Deleting a Project](#deleting-a-project)
+- [Project Rules](#project-rules)
+  - [Listing Rules for a Project](#listing-rules-for-a-project)
+  - [Getting a Project Rule](#getting-a-project-rule)
+  - [Creating a Project Rule](#creating-a-project-rule)
+  - [Updating a Project Rule](#updating-a-project-rule)
+  - [Deleting a Project Rule](#deleting-a-project-rule)
+- [Users](#users)
+  - [Listing Users](#listing-users)
+  - [Getting a User](#getting-a-user)
+  - [Creating a User](#creating-a-user)
+  - [Updating a User](#updating-a-user)
+  - [Deleting a User](#deleting-a-user)
+  - [List Teams for a User](#list-teams-for-a-user)
+  - [Updating Your Own User](#updating-your-own-user)
+- [Teams](#teams)
+  - [Listing Teams](#listing-teams)
+  - [Getting a Team](#getting-a-team)
+  - [Creating a Team](#creating-a-team)
+  - [Updating a Team](#updating-a-team)
+  - [Deleting a Team](#deleting-a-team)
+- [Team Membership](#team-membership)
+  - [Listing Team Users](#listing-team-users)
+  - [Adding Users to a Team](#adding-users-to-a-team)
+  - [Removing Users from a Team](#removing-users-from-a-team)
+- [Tokens](#tokens)
+  - [Listing Tokens](#listing-tokens)
+  - [Getting a Token](#getting-a-token)
+  - [Creating a Token](#creating-a-token)
+  - [Updating a Token](#updating-a-token)
+  - [Deleting a Token](#deleting-a-token)
+- [Restoring Admin Access](#restoring-admin-access)
+
+<!-- /TOC -->
+
+## General Notes
+
+1. If you do not have one already, generate an admin token for IAM v2.
+   (Neither The v1 command `chef-automate admin-token` nor any v1 admin tokens will work for v2.)
+   Use this, filling in a token name of your choice:
 
    ```bash
-   export TOKEN=`chef-automate iam token create TEST_ADMIN --admin`
+   export TOKEN=`chef-automate iam token create <your-token-name-here> --admin`
    ```
 
-## Policies
+2. URIs are relative to `https://<your-domain-here>/apis/iam/v2beta`, unless otherwise noted.
 
-### Listing Policies
+3. Attach the `?pretty` query string to an endpoint to get pretty-printed output.
+
+Putting it all together with the `/policies` endpoint as an example, this command fetches the list of policies with multi-line, formatted output:
 
 ```bash
-curl -s -H "api-token: $TOKEN" https://{{< example_fqdn "automate" >}}/apis/iam/v2beta/policies?pretty
+curl -sH "api-token: $TOKEN" \
+  https://{{< example_fqdn "automate" >}}/apis/iam/v2beta/policies?pretty
 ```
 
-The output includes both `Chef-managed` and `custom` policies.
-Policies migrated from IAM v1 are included as `custom` policies.
-Policy names that begin with `[Legacy]` are default policies migrated from v1.
-We recommend deleting these legacy policies and setting up new IAM v2 policies in their place.
-
-In the browser: Navigate to the *Policies* page under the **Settings**  tab.
+For those API methods that take JSON data (typically `create` and `update` methods)
```

Comment:
@msorens, I'm curious about the deleted information, here. I don't see it in the issue, but maybe I'm missing something?

Reply:
Yep, I see you found it had moved. The diff is a poor helper in this PR.